.evergreen/config.yml

```diff
@@ -157,37 +158,69 @@ axes:
       - id: "5.3"
         display_name: "Swift 5.3"
         variables:
-          SWIFT_VERSION: "5.3"
+          SWIFT_VERSION: "5.3.3"
+      - id: "5.4"
+        display_name: "Swift 5.4"
+        variables:
+          SWIFT_VERSION: "5.4.2"
+      - id: "5.5-dev"
+        display_name: "Swift 5.5-dev"
+        variables:
+          SWIFT_VERSION: "DEVELOPMENT-SNAPSHOT-2021-04-26-a"
+
+  - id: sanitize
+    display_name: Sanitize
+    values:
+      - id: tsan
+        display_name: TSan
+        variables:
+          SANITIZE: "thread"
+      # ASan is unfortunately known to produce false positives in Swift, see
+      # https://forums.swift.org/t/test-for-memory-leaks-in-ci/36526/9 
+      # - id: asan
+      #   display_name: ASan
+      #   variables:
+      #     SANITIZE: "address"
 
 buildvariants:
   - matrix_name: "tests-all"
     display_name: "${swift-version} ${os-fully-featured}"
     matrix_spec:
       os-fully-featured: "*"
       swift-version:
-        - "5.1"
         - "5.2"
+        - "5.3"
+        - "5.4"
+        - "5.5-dev"
     tasks:
       - name: "test"
 
-# define a separate matrix for 5.3 - no way to remove 5.3 + macOS from the definition above.
-# see EVG-13092
-  - matrix_name: "tests-all-5.3"
+  - matrix_name: "tests-5.1"
```

Comment:
do we want to do the compile-only setup on 5.1 here too?

Reply:
oh, I meant to add a comment about this. I thought about this, but the entire suite of tests for this library only take 2.5s seconds to run (for me locally anyway), so the savings are pretty minimal and I wasn't sure it was worth adding the complexity to our Evergreen setup.

Reply:
Ah okay, sounds good.